komodo/matrix.py

```diff
@@ -14,7 +14,7 @@ def get_matrix(
     current release matrix."""
     for product in itertools.product(rhel_versions, py_versions):
         rh_ver, py_ver = product
-        yield (f"rhel{rh_ver}", f"py{py_ver.replace('.', '')}")
+        yield (f"rhel{rh_ver}", f"py{str(py_ver).replace('.', '')}")
```

Comment:
is `str()` necessary here? `py_versions` is typed as `Sequence[str]` and `mypy` should locate any abuses of that (not having checked to which degree we enforce mypy here)

Reply:
It threw an exception that the replace method did not exist on type float, so I casted it to make sure it was a string.